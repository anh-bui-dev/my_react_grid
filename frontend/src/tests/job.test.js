import { shallow, mount } from 'enzyme';
import React, { Component } from 'react';
import Job from '../components/job';

describe('Job', () => {
  it('Should render correctly', () => {
    const detail = {};
    const wrapper = shallow(<Job detail={detail} />);
    expect(wrapper).toMatchSnapshot();
  });
});