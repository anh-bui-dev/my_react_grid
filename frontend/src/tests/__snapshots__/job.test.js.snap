// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should render correctly 1`] = `
<App>
  <div
    id="container"
  >
    <div
      className="author"
    >
      Jimmy Bui
    </div>
    <Jobs>
      <Loading>
        <div
          className="modal-mask"
        >
          <div
            className="loading d-flex justify-content-center"
          >
            <div
              className="spinner-grow text-secondary"
              role="status"
            >
              <span
                className="sr-only"
              >
                Loading...
              </span>
            </div>
          </div>
        </div>
      </Loading>
    </Jobs>
  </div>
</App>
`;

exports[`Job Should render correctly 1`] = `
<div>
  <Bootstrap(Table)
    borderless={true}
  >
    <tbody>
      <tr>
        <td
          className="label"
          colSpan={1}
        >
          Title:
        </td>
        <td
          colSpan={5}
        />
      </tr>
      <tr>
        <td
          className="label"
        >
          Date:
        </td>
        <td>
          10/09/2019
        </td>
        <td>
             
        </td>
        <td>
             
        </td>
        <td
          className="label"
        >
          Location:
        </td>
        <td />
        <td>
             
        </td>
        <td>
             
        </td>
      </tr>
      <tr>
        <td
          className="label"
          colSpan={1}
        >
          Applicants:
        </td>
        <td
          colSpan={5}
        />
      </tr>
      <tr>
        <td
          className="label"
          colSpan={1}
        >
          Description:
        </td>
        <td
          colSpan={5}
        />
      </tr>
    </tbody>
  </Bootstrap(Table)>
</div>
`;
